TopNews/app/src/main/java/com/fisfam/topnews/MainActivity.java

```diff
@@ -89,6 +96,35 @@ public boolean onOptionsItemSelected(MenuItem item) {
         return super.onOptionsItemSelected(item);
     }
 
+    //Choose country of news source
+    private void chooseCountry() {
+        mUserPref = new UserPreference(getApplicationContext());
+        // setup the alert builder
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle("Choose a country:");
```

Comment:
you should never hardcoded any string. Create a string value in resource because in real development, you would want to translate the string.